decode.go

```diff
@@ -1184,3 +1285,61 @@ func findEntryByKey(cur *yaml.Node, key string) (*yaml.Node, *yaml.Node, bool) {
 	}
 	return nil, cur, false
 }
+
+var nanosPerSecond = new(big.Int).SetUint64(uint64(time.Second / time.Nanosecond))
+
+var nanosMap = map[string]*big.Int{
+	"ns": new(big.Int).SetUint64(1),
+	"us": new(big.Int).SetUint64(uint64(time.Microsecond / time.Nanosecond)),
+	"µs": new(big.Int).SetUint64(uint64(time.Microsecond / time.Nanosecond)), // U+00B5 = micro symbol
+	"μs": new(big.Int).SetUint64(uint64(time.Microsecond / time.Nanosecond)), // U+03BC = Greek letter mu
+	"ms": new(big.Int).SetUint64(uint64(time.Millisecond / time.Nanosecond)),
+	"s":  new(big.Int).SetUint64(uint64(time.Second / time.Nanosecond)),
+	"m":  new(big.Int).SetUint64(uint64(time.Minute / time.Nanosecond)),
+	"h":  new(big.Int).SetUint64(uint64(time.Hour / time.Nanosecond)),
+}
+
+var unitsNames = []string{"h", "m", "s", "ms", "us", "ns"}
```

Comment:
Does this need `"µs"` too?

Reply:
that is the same as "us", this a normalized list. What do you think the default should be?

Reply:
(this will eventually be used to 'pretty' print durations)

Reply:
Oh in that case `us` definitely. Should not require anyone to pull out non-ASCII characters if they can be avoided.